test/integration/start_stop_delete_test.go

```diff
@@ -190,7 +216,7 @@ func validateEnableAddonAfterStop(ctx context.Context, t *testing.T, profile str
 	}
 
 	// Enable an addon to assert it comes up afterwards
-	rr, err := Run(t, exec.CommandContext(ctx, Target(), "addons", "enable", "dashboard", "-p", profile))
+	rr, err := Run(t, exec.CommandContext(ctx, Target(), "addons", "enable", "dashboard", "-p", profile, "--images=MetricsScraper=k8s.gcr.io/echoserver:1.4"))
```

Comment:
why do we change the image in both Addons metric-server in validateEnableAddonWhileActive and also change existing dasbhoard to custom image too ? if no good reason lets keep existing test like it was before.

Reply:
The dashboard addon now tests 2 things when minikube is stopped - that the Dashboard deployment properly starts (existing behavior), and that the MetricsScraper tries to load the custom image - notice that we only change the image for MetricsScraper and not for Dashboard. The distinction between this and the metric-server is that the metric-server is being enabled WHILE minikube is on. This caused a bug in my original implementation of this so I'd like to protect us from that in the future. Also I used different addons to ensure their settings don't collide in some strange way.